sprout/appliances/utils/providers.py

```diff
@@ -0,0 +1,71 @@
+from collections import Mapping
+
+import six
+
+from cfme.common.provider import all_types
+from cfme.exceptions import UnknownProviderType
+from cfme.utils import conf
+from cfme.utils.log import logger
+from sprout.vault.vault import settings
+
+providers_data = conf.cfme_data.get("management_systems", {})
```

Comment:
sprout is deployed along with test framework. do we really need to add it's own providers and vault files ?